python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -964,7 +964,7 @@ def aten〇embedding_bag〇padding_idx(weight: List[int], indices: List[int], of
     output_bag_shape.append(out_dim1)
 
     offset2bag_shape: List[int] = []
-    if mode == 1:
+    if mode == 0:
```

Comment:
Can you add an e2e test that results in a failure without this change and works with the change?

Reply:
umm , for the the mean mode would have to work. Since the mean mode == 1. 
The Sum mode == 0. 
I could implement the mean mode too ?

Reply:
I don't think the mean mode is needed. The sum mode will now go into an `if` that it was not going into before. I would expect this to be a behavior that can be tested using just the sum mode